packages/libraries/async-graphql/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "graphql-hive-async-graphql-extension"
+version = "0.0.1"
+edition = "2021"
+license = "MIT"
+
+[features]
+default = ["axum"]
+axum = ["dep:axum-core", "dep:http"]
+
+[dependencies]
+async-graphql = { git = "https://github.com/XiNiHa/async-graphql.git", rev = "16785021140660f8d62b85c17fec51428f4eb523" }
```

Comment:
I guess we can also just land it with your fork temporarily until the blocked PR is merged, cc @dotansimha @kamilkisiela